### ISSUE TITLE
Markers for application location on the map have disappeared

### ISSUE THREAD

**Issue by equivalentideas** [openaustralia/planningalerts#899]
The markers on the maps are really important to help you understand precisely where the application you're looking at is located. Something has happened, possible a change to the API we use for these maps, and now there are no markers.

In the words of the wonderful person who reported this bug:

> In Recent months, the markers that identified the actual site have disappeared from both the street map and the satellite maps.  Other than get in the car and visit the site, you can only guess and try and work out from the street directory, which property is actually the number in the application. 
 
> Is there a reason for this and can the essential markers be reinstated?? 

Here's on old screen shot from September 2015:
![screen shot 2015-09-11 at 6 01 57 pm](https://cloud.githubusercontent.com/assets/1239550/9809859/3ccc4536-58af-11e5-8284-934e8c84feb5.png)

Here's today (2016-02-10):
![screen shot 2016-02-10 at 10 10 32 am](https://cloud.githubusercontent.com/assets/1239550/12933852/8d27c586-cfde-11e5-8af2-400ff23db6c7.png)

Notice that the marker in the middle of the map is present in the first image and gone in the second.

**Comment by equivalentideas** [comment]
This is a bug consistent across applications.

**Comment by henare** [comment]
When Luke asked the person reporting the problem to confirm this was the issue, this is what they said:

>That's it exactly.  The fat, red exclamation mark with the black eye in the middle!  That's what we need back again.

I'm adding it here because I think it's so great :smile:

**Comment by equivalentideas** [comment]
Having trouble identifying what the problem is. I've just tried updating the [MXN version up a point](https://github.com/mapstraction/mxn/wiki/Changelog#v21-release-candidate-2), but that hasn't made a difference.

Looking into the Google API Side now.

**Comment by equivalentideas** [comment]
Not making any headway on this so I'm gonna put it down for today.

Here's some notes from where this is up to:

In the Google Maps API says you need to set the `map` attribute on the Marker to get it put into the map. [Mxn](http://mapstraction.com/mxn/build/latest/docs/symbols/mxn.Marker.html#show) does handle this for us [when we add our marker to the map](https://github.com/openaustralia/planningalerts/blob/master/app/assets/javascripts/maps/maps.js#L9).

There doesn't seem to be any changes in the Google Maps api that would impact the marker https://developers.google.com/maps/documentation/javascript/releases#320 .

**Comment by henare** [comment]
Legend :metal:

**Comment by equivalentideas** [comment]
Using `git bisect` I've tracked the change down to this commit ab680c6dbd526afc193e6b58be475475052c5728 . I had been a bit silly and introduced a global helper/utility class for hiding things, rather than just targeting my specific use case. This interacted badly with Google Maps as they're using `aria-hidden` on content that is visible for people not using assistive technology. Silly me.

I've fixed this in aa09ad5
